spec/system/forgotten_password_flash_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe 'Comfirmed user, following forgotten password process', type: :system do
+  let(:user) { create :user, :confirmed }
+  let(:token) { user.send_reset_password_instructions }
+
+  context 'when resetting password' do
+    it 'flash message displays correctly' do
+      visit "/users/password/edit?reset_password_token=#{token}"
+      fill_in 'New password', with: 'abc123'
+      fill_in 'Confirm your password', with: 'abc123'
+      click_on "Reset password"
+      expect(page).to have_selector ".govuk-notification-banner--success"
```

Comment:
@ebrett @adeoke I think what we want is to just assert the text in this spec and none of the classes.
Which is where we let our page model UI framework do the work. Do you agree?
cc @timallen-and

Reply:
That is generally what I do, i.e. just test for the "Success" text and the body of the message.  I would sometimes use the selector to test that the text was in that particular part of the page, but SitePrism handles that pretty seamlessly, so I agree.